src/display/editor/stamp.js

```diff
@@ -311,6 +311,12 @@ class StampEditor extends AnnotationEditor {
       this.parent.addUndoableEditor(this);
       this.#hasBeenAddedInUndoStack = true;
     }
+    this._uiManager.reportTelemetry({
+      type: "editing",
+      data: {
+        type: "image",
+      },
+    });
```

Comment:
To help me understand how this all "fits together", what's the different between this new one and the existing "stamp" one that's already reported in the viewer?
https://github.com/mozilla/pdf.js/blob/b80e0d881dc9b3282b02acd2fe7feadd7c25bde6/web/app.js#L1769-L1774

Maybe add a (short) comment explaining why we need this separate one here?